modules/samples/sample-clients/ei-analytics-client/src/main/java/org/wso2/carbon/sample/eianalytics/Client.java

```diff
@@ -76,9 +76,19 @@ public static void main(String[] args) {
             Event flowEntryEvent = new Event();
             flowEntryEvent.setStreamId(DataBridgeCommonsUtils.generateStreamId(FLOW_ENTRY_STREAM_NAME, VERSION));
             flowEntryEvent.setCorrelationData(null);
-            for (int i = 0; i < noOfBatches; i++) {
+            while (true) {
```

Comment:
Why can't we have `while(!finished)` and get rid of 90-92/?

Reply:
Added the changes.